modules/react/dialog/stories/Dialog.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import {SymbolDoc, Specifications} from '@workday/canvas-kit-docs';
+
+import {Dialog} from '@workday/canvas-kit-react/dialog';
+import {Basic} from './examples/Basic.tsx';
+import {Focus} from './examples/Focus.tsx';
+
+<Meta title="Components/Popups/Dialog" component={Dialog} />
+
+# Canvas Kit Dialog
+
+A Dialog component is a non-Modal type of Dialog that will not render the rest of the page inert
+while it is active. A Dialog should be used in situations where the task is not critical.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Usage
+
+### Basic
```

Comment:
Should include "Example." I see the Modal docs don't do this properly for the Basic Example (we can fix that later).

```suggestion
### Basic Example
```